packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -149,13 +150,29 @@ module.exports = {
       'react-native': 'react-native-web',
     },
     plugins: [
+      // Adds support for installing with Plug'n'Play, leading to faster installs and adding
+      // guards against forgotten dependencies and such.
+      PnpWebpackPlugin,
       // Prevents users from importing files from outside of src/ (or node_modules/).
       // This often causes confusion because we only process files within src/ with babel.
       // To fix this, we prevent you from importing files out of src/ -- if you'd like to,
       // please link the files into your node_modules/ and let module-resolution kick in.
       // Make sure your source files are compiled, as they will not be processed in any way.
       new ModuleScopePlugin(paths.appSrc, [paths.appPackageJson]),
     ],
+    // Plug'n'Play relies on symlink for its virtual paths (ie peer dependencies), which Webpack
+    // always resolve to the absolute path on disk by default.
+    symlinks: false,
```

Comment:
If we do this, symlinking `src/` to `node_modules/@` won't work because we'll treat app code as `node_module` code (instead of application code).
We could add special behavior to compile code from `@` as app code, but I'd want to collect feedback on what people are using symlinks for currently -- I suspect it'll be in line with the behavior we want.

Reply:
@Timer where is defined this `@` symlink? Shouldn't it be an `alias` entry in webpack/jest?

Reply:
We didn’t add this feature yet. We were hoping we could do it via symlink instead of adding a special alias to every tool. It would be nice to not confuse IDEs, Flow, etc, which symlink could help with.